persistence/inmemory.go

```diff
@@ -17,6 +17,11 @@ func NewInMemoryStore(defaultExpiration time.Duration) *InMemoryStore {
 	return &InMemoryStore{*cache.New(defaultExpiration, time.Minute)}
 }
 
+// TODO: stub func
+func (c *InMemoryStore) Mget(values []interface{}, keys ...string) error {
```

Comment:
Same thing: why add this to the interface, especially if you're not implementing it in all the concrete types?